### DISCUSSION TITLE
making a page responsive for mobile

### DISCUSSION THREAD

**Discussion by Fordp12** [twbs/bootstrap#35787] [Q&A]
Hi
I am making a website in Nationbuilder that uses bootstrap. Unfortunately the free templates are not mobile responsive and the page I have added does not display correctly on a mobile device.
I have three images which display horizontally on a desktop pc but they don't display vertically on a mobile device. They squash up horizontally. I have to turn my phone horizontal.

**Comment by Arhell** [comment]
https://getbootstrap.com/docs/5.1/getting-started/introduction/
in search input add **responsive**
